agent/file.py

```diff
@@ -0,0 +1,50 @@
+"""Collection of functions to handle files."""
+import requests
+import tenacity
+from ostorlab.agent.message import message as m
+
+DOWNLOAD_REQUEST_TIMEOUT = 60
```

Comment:
i think 60 is a lot for requests timeout !

Reply:
It is a safe value. Bigger has little impact.